File: fizzbuzz.js
const x = 15

for(var i = 1; i<x + 1;i++){
    let resp = ""
    if(i % 3 == 0){
        resp = resp + "Fizz"
    }
    if(i % 5 == 0){
        resp += "Buzz"
    }
    if(i % 3 != 0 && i % 5 != 0){
        resp = i
    }
    console.log(resp)
}

File: FizzBuzz.py
h = 15
for i in range(h + 1):
    resp = ""
    if i % 3 == 0:
        resp += "Fizz"
    if i % 5 == 0:
        resp += "Buzz"
    if i % 3 != 0 and i % 5 != 0:
        resp += str(i)
    if(i == 0):
        resp = "0"
    print(resp)
